Source/Venice/Channel/Channel.swift

```diff
@@ -67,26 +67,23 @@ public final class Channel<T>: Sequence {
     }
 
     /// Closes the channel. When a channel is closed it cannot receive values anymore.
-    public func close() -> Bool {
-        if closed {
-            return false
-        }
+    public func close() throws {
+        guard !closed else { return }
```

Comment:
Yeah it's some kind of stream. But it's not a binary one. `Stream` from `C7` represents a binary data stream. Which does not correlate to channels. This raises the question. Should we rename `Stream` to `BinaryStream` in `C7`? Or should we add the broader concept of `Stream` with an `associatedType` which defines the type of the `Stream`? I'm not very fond of `associatedType`s in protocols for Open Swift, cause they can overcomplicate things. But we should at least consider renaming?